app/views/courses/_course_table_row.html.erb

```diff
@@ -12,7 +12,7 @@
     </td>
   <% end %>
   <td class="govuk-table__cell" data-qa="courses-table__status">
-    <% unless course.not_running? %>
+    <% if course.is_running? || course.is_withdrawn? %>
```

Comment:
When the work is done to update the manually withdrawn courses, we can probably do away with this `if` statement.

Reply:
:+1: